ios/Classes/NativeBridge/Handlers/RegistrationHandler.swift

```diff
@@ -288,18 +251,20 @@ extension RegistrationHandler: ONGRegistrationDelegate {
         }
     }
     
-    private func mapErrorMessageFromStatus(_ status: Int) -> String? {
-        var errorMessage: String? = nil
-        
-        if status <= 2000 {
-            errorMessage = nil
-        } else if status == 4002 {
-            errorMessage = "This code is not initialized on portal."
-        } else {
-            errorMessage = "Provided code is incorrect."
+    private func setupCustomInfoResponse(_ challenge: ONGCustomRegistrationChallenge) -> Dictionary<String, Any?> {
```

Comment:
Rename to something like `makeCustomInfoResponse`

Reply:
done